src/schema/users.ts

```diff
@@ -1866,6 +1878,77 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         weekStart,
       };
     },
+    recoverStreak: async (
+      _,
+      __,
+      ctx: AuthContext,
+      info,
+    ): Promise<GQLUserStreak> => {
+      const { userId } = ctx;
+      const user = await ctx.con
```

Comment:
There's `@auth` directive on graphQL side we shouldn't need to manually verify again.

Reply:
I still need the `user.reputation` object below, should I change `findOneByOrFail` with `findOneBy`?

Reply:
Oh right, in that case leave it yes :)